util/rom-ext-manifest-generator.py

```diff
@@ -170,14 +204,14 @@ def generate_format_description(regions, output_dir):
 
     output_path = output_dir / 'manifest.txt'
 
-    truncate_length = 16 # bytes
+    truncate_length = 16  # bytes
     bits_in_byte = 8
 
     verbose_regions = []
```

Comment:
nit: This is not related to the new `verbose` flag, right? I know this was already named `verbose_regions` but IMHO using the `protocol_input` from the beginning or something like `protocol_regions` would be clearer.

Reply:
Yeah, this is probably a bad name. I'll do a small PR this afternoon to fix this.